src/Model/Item.php

```diff
@@ -72,7 +72,7 @@ class Item extends BaseObject
 			'title', 'content-warning', 'body', 'location', 'coord', 'app',
 			'rendered-hash', 'rendered-html', 'object-type', 'object', 'target-type', 'target',
 			'author-id', 'author-link', 'author-name', 'author-avatar',
-			'owner-id', 'owner-link', 'owner-name', 'owner-avatar'];
+			'owner-id', 'owner-link', 'owner-name', 'owner-avatar', 'emailcc'];
```

Comment:
There is no "emailcc" field in the item table. What is this change supposed to do?

Reply:
I have an E_NOTICE here in `mod/item.php`. I think @MrPetovan was not aware of it. So what should I do instead?

Reply:
Fix `mod/item.php` instead?

Reply:
I was trying to find the root cause of the bug. :-) And I thought that the array `$orig_post` returned by `Item::selectFirst()` may miss that element.